examples/operations.ipynb

```diff
@@ -0,0 +1,515 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These examples illustrate common operations on polyhedra using [Polyhedra.jl](https://github.com/JuliaPolyhedra/Polyhedra.jl):\n",
+    "\n",
+    "- [Intersection](#Intersection).\n",
+    "- [Convex hull](#Convex-hull)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "using Polyhedra, CDDLib"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Intersection <a name=\"Intersection\"></a> \n",
+    "\n",
+    "Intersection of polyhedra is obtained with the [`intersect`](https://juliapolyhedra.github.io/Polyhedra.jl/latest/utilities.html#Base.intersect) function. See also the wikipedia page: [Intersection (set theory)](https://en.wikipedia.org/wiki/Intersection_set_theory).\n",
+    "\n",
+    "Below we compute the intersection of two randomly generated polygons in V-representation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "P1 = polyhedron(SimpleVRepresentation(randn(15, 2)), CDDLibrary())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "P2 = polyhedron(SimpleVRepresentation(randn(15, 2)), CDDLibrary());"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Pint = intersect(P, P2);"
```

Comment:
`P` should be `P1`.

Reply:
> I am not sure about pushing the output.

i'm not sure to understand, is this to remove the `;` (there will be julia's output as in the REPL), and then  click "Restart & Clear output" to remove it? Ok! 

figures display well in github's online viewer (or nbviwer), i'm not aware of those issues.

> P should be P1.

yes!

i'll update the branch tomorrow, thanks for reviewing.

Reply:
If it displays well on Github then keep the output :)